load_clean.py

```diff
@@ -38,20 +43,39 @@ def clean_data(df):
     # Rename the 'Country' column to 'Country (GDP, current prices)'
     df = df.rename(columns={'GDP, current prices (Purchasing power parity; billions of international dollars)': 'Country'})
 
-    # Filter columns for the years 2000 to 2022, assuming the year columns exist as actual years
+    # Filter columns for the years 2000 to 2022
     year_columns = [str(year) for year in range(2000, 2023)]
-    
-    # Select only the relevant year columns'
+
+    # Select only the relevant year columns and 'Country' column
     country_column = 'Country'
-    
     columns_to_keep = [country_column] + year_columns
     df_clean = df[columns_to_keep]
 
     print(df_clean.head(8))
```

Comment:
Consider replace the print function with log function to help debug. Generally it is not a good practice to leave print in the function.

Reply:
Thanks for the tip, will look to implement